File: src/main/java/fr/vlegall/sochief/repository/UtensilRepository.java
package fr.vlegall.sochief.repository;

import fr.vlegall.sochief.model.recipe.Utensil;
import org.springframework.data.jpa.repository.JpaRepository;

public interface UtensilRepository extends JpaRepository<Utensil, Long> {
    Utensil findByNameIgnoreCase(String name);
}

File: src/main/kotlin/fr/vlegall/sochief/repository/UtensilRepository.kt
package fr.vlegall.sochief.repository

import fr.vlegall.sochief.model.recipe.Utensil
import org.springframework.data.jpa.repository.JpaRepository

interface UtensilRepository : JpaRepository<Utensil, Long> {
    fun findByNameIgnoreCase(name: String): Utensil?
}
